platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -75,6 +75,9 @@
   @Nullable
   private Style style;
 
+  @Nullable
+  private String localIdeographFontFamily;
```

Comment:
`OfflineRegionDefinition` is not tied to the map in any way and won't be able to automate this reference, the user will have to pass the boolean manually anyway, so storing and exposing it here seems unnecessary.

Reply:
:+1: